contracts/RiskManagerV1.sol

```diff
@@ -332,7 +332,8 @@ contract RiskManagerV1 is IRiskManagerV1, Auctioneer, Ownable {
     }
 
     /// @notice Begins the bond auction threshold update process.
-    /// @dev Can be called only by the contract owner.
+    /// @dev Can be called only by the contract owner. The value of the
+    ///      threshold should not be greater than 100.
     /// @param _newBondAuctionThreshold New bond auction threshold in percent
     function beginBondAuctionThresholdUpdate(uint256 _newBondAuctionThreshold)
```

Comment:
Here we are setting a percentage value.
We could consider checking `_newBondAuctionThreshold` (it should not be greater than 100).

Reply:
Let's do this.

Reply:
Done in https://github.com/keep-network/coverage-pools/pull/160/commits/db6530d23902d656cb1c244a71203c4be7077f3d.